pages/common/forever.md

```diff
@@ -0,0 +1,15 @@
+# forever - nodejs tool
+
+> Server-side JavaScript application that makes sure node applications run indefinitely (restarts after crash).
+
+- Start running a file forever (as a daemon):
+
+`forever {{file}}.js`
+
+- List forever running processes:
```

Comment:
Mention that IDs are also listed in the output, so that you don't have to mention - ID from "forever list" in the next example.